resources/views/whiteboards/draw.blade.php

```diff
@@ -12,6 +12,7 @@
             gallery="{{ route('gallery.browse', $campaign) }}"
             search="{{ route('search.live', $campaign) }}"
             entity="{{ route('entities.json.export', [$campaign, 0]) }}"
+            user="{{ auth()->user() ? auth()->user()->id : null }}"
```

Comment:
auth()->check() is the correct test, or simply `auth()->user()?->id ?? null` I think